extra/mason/report/report-tests.factor

```diff
@@ -29,5 +30,5 @@ mason.report mason.common mason.config namespaces tools.test xml xml.writer ;
         [ status-error ] [ 1236 test-failed ] unit-test
         verify-report
 
-    ] with-temp-directory
```

Comment:
I think we want to use a ``temp-directory`` to copy the fake data into, not the ``builds-dir``.